src/System.Private.ServiceModel/tests/Common/src/ServiceInterfaces.cs

```diff
@@ -47,6 +48,18 @@ public interface IWcfService
 
     [OperationContract(Action = "http://tempuri.org/IWcfService/GetRequestCustomHeader", ReplyAction = "*")]
     string GetRequestCustomHeader(string customHeaderName, string customHeaderNamespace);
+
+    [OperationContract]
+    Stream GetStream(string data);
+
+    [OperationContract]
+    bool UploadStream(Stream stream);
+
```

Comment:
A 'bool' return isn't very informative.  I recommend it return the stream as a string so the test can verify it was received correctly.